modules/virt-about-runstrategies-vms.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-runstrategies-vms_{context}"]
+= About RunStrategies for virtual machines
+
+A `RunStrategy` for virtual machines determines a virtual machine instance (VMI)'s behavior, depending on a series of conditions. The `RunStrategy` setting exists in the virtual machine configuration process as an alternative to the `spec.running` setting.
+The `RunStrategy` setting allows for greater flexibility for how VMIs are created and managed, in contrast to the `spec.running` setting with only `true` or `false` responses.
+
+There are four defined RunStrategies.
+
+* Always: A VMI is always present when a virtual machine is created. A new VMI is created if the original crashes.
```

Comment:
Add also:
"This is the same behavior as `spec.running: true`"

Reply:
Added as part of "Always" line